Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -1473,10 +1473,12 @@ public void UpdateResultView(ICollection<ResultsForUpdate> resultsForUpdates)
                     {
                         result.Score = int.MaxValue;
                     }
-                    else
+                    else if (result.Score != int.MaxValue)
```

Comment:
Just wanted to note here, at some point I think it's best to provide an actual max value- standardise the scoring system so to not allow randomly assigning any score by plugins. 

Also this int max value is not documented anywhere so unless devs look into this Result class they probably won't know that they can do this.

Any how just thought I mention these, but they are out side the scope of this change.

Reply:
@jjw24 Now I export the max score in the `Result` class so developers can see this maximum score.